File: 2000-reverse-prefix-of-word/2000-reverse-prefix-of-word.java
class Solution {
    public String reversePrefix(String word, char ch) {
        int f = word.indexOf(ch);
        if(f==-1)return word;
        StringBuilder sb = new StringBuilder(word);
        sb.delete(f+1,word.length());
        sb.reverse();
        return sb.toString()+word.substring(f+1);
    }
}

File: 2000-reverse-prefix-of-word/2000-reverse-prefix-of-word.py
class Solution(object):
    def reversePrefix(self, word, ch):
                idx = word.find(ch)
                if idx == -1:
                    return word
                idx2 = idx+1
                ans =""
                while(idx>-1):
                    ans+=word[idx]
                    idx-=1
                while(idx2<len(word)):
                    ans+=word[idx2]
                    idx2+=1
                return ans    
        